green/test/test_loader.py

```diff
@@ -264,6 +264,16 @@ def test_method2(self):
                          set(['test_method1', 'test_method2']))
 
 
+    def test_nose_disabled_attribute(self):
+        "Tests disabled by nose generators dont get loaded"
+        class HasDisabled(unittest.TestCase):
+            def test_method(self):
+                pass
+
+        HasDisabled.test_method.__test__ = False
```

Comment:
Oh weird, it was passing locally with python2.

I guess I'll just have to use `setattr`